react/features/base/util/uri.js

```diff
@@ -298,7 +314,9 @@ export function parseURIString(uri: ?string) {
     // The room (name) is the last component of pathname.
     const { pathname } = obj;
 
-    obj.room = pathname.substring(pathname.lastIndexOf('/') + 1) || undefined;
+    obj.room = _normalizeRoomName(
+        pathname.substring(pathname.lastIndexOf('/') + 1)
+    );
```

Comment:
1. That's not how we break function calls into multiple lines. We do something like:
```javascript
obj.room
    = _normalizeRoomName(pathname.substring(pathname.lastIndexOf('/') + 1));
```

Then we break at the function arguments (if necessary):
```javascript
obj.room
    = _normalizeRoomName(
        pathname.substring(pathname.lastIndexOf('/') + 1),
        secondArgument,
        thirdArgument,
        etc);
```

Then we move further in the same patter (if necessary):
```javascript
obj.room
    = _normalizeRoomName(
        pathname.substring(
            pathname.lastIndexOf('/') + 1),
        secondArgument,
        thirdArgument,
        etc);
```

2. The original expression `pathname.substring(pathname.lastIndexOf('/') + 1) || undefined` meant that, if `room` happens to be the empty string, we'd rather represent the lack of a room name with `undefined`.

Reply:
Fixed, thanks!